PayoutCurve.md

```diff
@@ -69,7 +69,7 @@ In this section we detail the TLV serialization for a general `payout_function`.
 
 1. type: 42790 (`payout_function_v0`)
```

Comment:
I think this needs to be updated as I don't believe its a TLV in the #163 test vectors?

Reply:
Thanks! Fixed everything in that file, as well as the `RoundingIntervals` that was also not updated.